allennlp/training/trainer.py

```diff
@@ -176,6 +176,28 @@ def __call__(
 EpochCallback.register("null")(EpochCallback)
 
 
+@EpochCallback.register("track_epoch_callback")
+class TrackEpochCallback:
+    """
+    A callback that you can pass to the `GradientDescentTrainer` to access the current epoch number
+    in progress in the model as a class member `model.epoch`. Since the EpochCallback passes `epoch=-1`
```

Comment:
Minor wording suggestion:

```suggestion
    A callback that you can pass to the `GradientDescentTrainer` to access the current epoch number
    in your model during training. This callback sets `model.epoch`, which can be read inside of
    `model.forward()`. Since the EpochCallback passes `epoch=-1`
```

Reply:
Github let me just commit this myself, so I did it.